pages/product/product.html

```diff
@@ -0,0 +1,240 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/617c60ee1d.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="https://artsyui.netlify.app/src/artsyUI.css" />
+    <link rel="stylesheet" href="../../style.css" />
+    <link rel="stylesheet" href="./product.css" />
+    <title>Artsy-shop</title>
+  </head>
+
+  <body class="product_page-body">
+    <!--********** Header ************-->
+    <header class="arstyShop-header product_page-header">
+      <h3>Artsy Shop</h3>
+
+      <div class="artsy-input icon-input">
+        <i class="fas fa-search"></i>
+        <input placeholder="Search..." type="text" />
+      </div>
+
+      <div class="right-side">
+        <a href="">
+          <button class="btn btn-primary">Login</button>
+        </a>
+        <div onclick="openLogin()">
+          <i class="fa-regular fa-user"></i>
+        </div>
+
+        <a class="favourite" href="">
+          <i class="fa-regular fa-heart"></i>
+          <span class="items">25</span>
+        </a>
+
+        <a class="cart" href="">
+          <i class="fa-solid fa-bag-shopping"></i>
+          <span class="items">25</span>
+        </a>
+      </div>
+    </header>
+
+    <!--********** aside ************-->
+    <!-- <aside class="product_page-aside">
+      <div class="">
+        <h3>Price</h3>
+        
+      </div>
+      <div class="aside-sec">
+        <h3>Category</h3>
+        <label>
+          <input type="checkbox" /> 
+          <span class="checkbox-text">Abstraction</span>
+        </label>
+        <label><input type="checkbox" /> 
+          <span class="checkbox-text">Nature</span>
+        </label>
+        <label><input type="checkbox" /> 
+          <span class="checkbox-text">Landscape</span>
+        </label>
+      </div>
+      <div class="aside-sec">
```

Comment:
Great work on the UI but the hover on sort by could be made better. refer to these images:
![image](https://user-images.githubusercontent.com/67316487/153817122-882d2799-e0f6-441b-ab59-d107aec6c7ce.png)
![image](https://user-images.githubusercontent.com/67316487/153817196-4705f969-5310-4400-8dde-4042079ab616.png)